### PR TITLE
Add new review bot that checks that request descriptions contain certain tags (bnc#, boo#, etc.)

### PR DESCRIPTION
**Pull request by antlarr**
Add new review bot that checks that request descriptions contain certain tags (bnc#, boo#, etc.) and also a test script that tests different type of requests including and not including some of those tags.

### CODE REVIEW

File: tests/checktags_tests.py [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+        self.assertFalse(result['state_accepted'])
+
+        # second time request is in state new so we can accept
+        # first time request is in in review
+        self.checker.set_request_ids(['770001'])
+        self.checker.check_requests()
+
+        self.assertFalse(result['state_accepted'])
+
+    def test_non_maintainer_submit(self):
```
Comment (coolo) [inline tests/checktags_tests.py]: kind of obvious what you copied from :)

**Comment by coolo** [comment]
The request description is not what we care about - we want to check the (possibly many) .changes file diff.

File: tests/checktags_tests.py (hunk 1 above)
Comment (antlarr) [inline tests/checktags_tests.py]: hehe, never tried to hide it. I copied and adapted some tests from maintenance_tests and factory_source_tests :)

**Comment by antlarr** [comment]
ok, I'm working on the .changes file(s) diff now

**Comment by antlarr** [comment]
aplanas, thanks for the suggestion. It's nicer with super indeed. I already added it but will commit it tomorrow morning together with the modifications to check the tags in the .changes files diff instead of the request description, which I've just finished implementing.

**Comment by coolo** [comment]
LGTM

**Comment by coolo** [comment]
https://build.suse.de/request/show/53266 uses "bnc #NR" and that should be acceptable too I guess. At least OBS parses it.

**Comment by coolo** [comment]
on the other hand our documentation is clear: "no whitespace between prefix and number"

**Pull request merged**